gdpr/utils.py

```diff
@@ -0,0 +1,16 @@
+from users.models import User
+
+
+def get_user(user: User) -> User:
+    """
+    Get the user provider, as User is the GDPR API root model. GDPR API implementation
+    by default attempts to get it from the root models user attribute.
+    """
+    return user
+
+
+def delete_user_data(user: User, dry_run: bool) -> None:
```

Comment:
is the `dry_run` parameter not used?

Reply:
Is it automatically detected and implented without writing any code ourself? Based on the helusers GDPR repo:
> The function gets called within a database transaction, which gets automatically rolled back if it's a dry run operation.

Reply:
This is another thing that the package kind of hides. It uses exceptions to handle this in the views `delete` method.
Do you think it would be more clear in our code, if we raised the same exception in `delete_user_data()` when `dry_run is True`? (Or maybe rather is not False)